docker-compose.yml

```diff
@@ -93,7 +93,7 @@ services:
   smtp:
     <<: *restart_policy
     image: tianon/exim4
-    hostname: ${SENTRY_MAIL_HOST:-''}
+    hostname: ${SENTRY_MAIL_HOST:-}
```

Comment:
lgtm

> Additionally when using the [2.1 file format](https://docs.docker.com/compose/compose-file/compose-versioning/#version-21), it is possible to provide inline default values using typical shell syntax:
>
> - `${VARIABLE:-default}` evaluates to `default` if `VARIABLE` is unset or empty in the environment.

https://docs.docker.com/compose/environment-variables/

I'll take your word for it on quoting, @glensc. :P

Reply:
No need to take on anything, just see the proof in PR body.